projects/vdk-plugins/.plugin-common.yml

```diff
@@ -27,7 +27,7 @@
   artifacts:
     when: always
     reports:
-      junit: tests.xml
+      junit: "projects/vdk-plugins/$PLUGIN_NAME/tests.xml"
```

Comment:
may you link a pipeline before/after?

nit: the PR description does not seem to match the draft changes 👀

Reply:
It's a draft . I am still working on the change. And I needed to test using the Gitlab CI since I cannot locally as easily for this. 

Anyhow I opened a PR for the change since it should be separate from this one : https://github.com/vmware/versatile-data-kit/pull/1732